uiowa/uids
# Add brand bar and logo components, clean-up

Resolves #691 
Resolves #692 

# To test
- Review[ code changes](https://github.com/uiowa/uids/pull/694/files)
- Review https://uids.brand.uiowa.edu/branches/iowa_bar/ to see Brand Bar and Logo components. Make note of things that seem off.
- Note that test passed below :)
